hieradata/clients/azure.ci.jenkins.io.yaml

```diff
@@ -328,15 +328,18 @@ profile::jenkinscontroller::jcasc:
             spot: true
             userData: &BootstrapDatadogScript
               - "#!/bin/bash"
+              - "set -eux"
+              - echo "START CLOUDINIT"
```

Comment:
```suggestion
              - echo "START CLOUDINIT"
              - systemctl stop datadog-agent.service
              - mkdir -p /var/log/datadog /etc/datadog-agent
```

Reply:
- Start the datadog service (just in case if it is started) as a preliminary to the configuration setup
- Ensure that the 2 directories are created, also as a preliminary